routes/user.js

```diff
@@ -16,6 +16,7 @@ router.get("/verify", jsonParser, asyncMiddleware(verify));
 router.use("/unsubscribe", urlEncodedParser);
 router.get("/unsubscribe", asyncMiddleware(getUnsubscribe));
 router.post("/unsubscribe", asyncMiddleware(postUnsubscribe));
-
+router.get("/unsubscribe_survey", asyncMiddleware(getUnsubSurvey));
```

Comment:
Doesn't need `asyncMiddleware` if/when we remove the `async` from `getUnsubSurvey`